CMakeLists.txt

```diff
@@ -14,14 +14,19 @@ PROJECT(Descent3)
 IF (UNIX)
 SET (D3_GAMEDIR "~/Descent3/")
 
+  IF (APPLE)

```

Comment:
I doubt that in Apple there may 64bit code generated. Also, -m32 should be added via compiler toolchain, not directly in CMake project file.

Reply:
Yes, `-m32` did not quite work, removed entirely.